mathics/builtin/base.py

```diff
@@ -362,7 +362,7 @@ def get_option_string(self, *params):
         return None, s
 
 
-class InstancableBuiltin(Builtin):
+class InstanceableBuiltin(Builtin):
```

Comment:
This name feels a bit weird to me.  In a PR off of this I might have changed the name. I don't remember. Right now I can't come up with another name.

Reply:
Ok. I can't come up with another name and it problably isn't that important either. 

Sorry for the delay - thanks for your patience.